google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/cloudpubsub/PublisherSettings.java

```diff
@@ -125,16 +188,12 @@ Publisher instantiate() throws ApiException {
                   SinglePartitionPublisherBuilder.Builder singlePartitionBuilder =
                       underlyingBuilder()
                           .setBatchingSettings(batchingSettings)
-                          .setContext(PubsubContext.of(FRAMEWORK))
```

Comment:
Beam also touches this removed setContext method. Should it just remove the usage?

Reply:
This has been moved to newServiceClient method. (Thanks @dpcollins-google !)